<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>instrumentsWindow</class>
 <widget class="QMainWindow" name="instrumentsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="TreeWidget" name="tree">
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Value</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Address / Parameters</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Status</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hbox_device">
      <item>
       <widget class="QPushButton" name="pb_get">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Get value</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_set">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Set value</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_rename">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Rename</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_config">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Config device</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>813</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuImport">
    <property name="title">
     <string>Import</string>
    </property>
    <addaction name="actionfrom_JSON"/>
   </widget>
   <widget class="QMenu" name="menuExport">
    <property name="title">
     <string>Export</string>
    </property>
    <addaction name="actionto_JSON"/>
    <addaction name="actionto_PyHegel_script"/>
   </widget>
   <addaction name="menuImport"/>
   <addaction name="menuExport"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad"/>
   <addaction name="actionConfig"/>
   <addaction name="separator"/>
   <addaction name="actionAdd"/>
   <addaction name="actionRemove"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionAdd">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../resources/list-add.svg</normaloff>../resources/list-add.svg</iconset>
   </property>
   <property name="text">
    <string>Import instrument</string>
   </property>
   <property name="toolTip">
    <string>Add an instrument</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../resources/list-remove.svg</normaloff>../resources/list-remove.svg</iconset>
   </property>
   <property name="text">
    <string>Remove instrument</string>
   </property>
   <property name="toolTip">
    <string>Remove selected instrument</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../resources/load.svg</normaloff>../resources/load.svg</iconset>
   </property>
   <property name="text">
    <string>(re)load</string>
   </property>
   <property name="toolTip">
    <string>Load (reload) instrument</string>
   </property>
  </action>
  <action name="actionto_JSON">
   <property name="text">
    <string>to JSON</string>
   </property>
  </action>
  <action name="actionto_PyHegel_script">
   <property name="text">
    <string>to PyHegel script</string>
   </property>
  </action>
  <action name="actionfrom_JSON">
   <property name="text">
    <string>from JSON</string>
   </property>
  </action>
  <action name="actionConfig">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../resources/config.svg</normaloff>../resources/config.svg</iconset>
   </property>
   <property name="text">
    <string>config</string>
   </property>
   <property name="toolTip">
    <string>Show instrument config window (if it exists)</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>widgets/TreeWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
